FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_basic_tls/DemoTasks/BasicTLSMQTTExample.c

```diff
@@ -0,0 +1,877 @@
+/*
```

Comment:
I think this file can be more appropriately named `ServerAuthMQTTExample.c` as the Mutual Auth demo is named `MutualAuthMQTTExample.c`

Reply:
I agree, but I just confirmed that for the sake of consistency with the other products, it would be better to maintain the "Basic TLS" convention. Hopefully it can be changed in the future.